docs/UserGuide.md

```diff
@@ -339,6 +341,13 @@ Exports a CSV file of the current view containing specified fields.
 
 Notes:
 * Name is always the first column in the CSV file.
+* Including any of the arguments will export include that field in the CSV file. 
+* Each argument represents: 
```

Comment:
Do we need to state this here? If you want perhaps putting it at the top will be better since its used for other commands too.

Reply:
Thats true but I was thinking would be good to include a refresher